package.json

```diff
@@ -9,66 +9,8 @@
   "devDependencies": {
     "phonegap": "7.1.1"
   },
-  "cordova": {
-    "platforms": [
-      "ios",
-      "android"
-    ],
-    "plugins": {
-      "phonegap-plugin-push": {
```

Comment:
and package.json?

Reply:
I don't know why they were removed. I added them back in the most recent commit, and it still works while running locally.